docs/core/integration-quality-scale/rules/iqs038.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "IQS038 - Make effective use of the config entry data and options"
```

Comment:
I don't think this should be a special rule. Just add it to the config flow rule on how to do the config flow.